### PR TITLE
CSV data loads nice

### PR DESCRIPTION
**Pull request by joemecha**
To load CSV files:
1. Added missing column to invoice_items and changed credit card number to string
2. Created models for each object type (e.g. customer)
3. Read logic added to db/seeds

Please review!

### CODE REVIEW

**Review by IcelandicGambit — COMMENTED** [review]
it does load nice

File: db/migrate/20210413020416_create_transactions.rb [hunk 1]
```diff
@@ -2,7 +2,7 @@ class CreateTransactions < ActiveRecord::Migration[5.2]
   def change
     create_table :transactions do |t|
       t.integer :invoice_id
-      t.integer :credit_card_number
+      t.string :credit_card_number
```
Comment (ghost) [inline db/migrate/20210413020416_create_transactions.rb:5]: Agree that using a string here is best. Read some articles that you should default to a string for any numbers that you don't need to do calculations with, but will be interesting to see what we learn here.

File: db/seeds.rb [hunk 2]
```diff
@@ -1,11 +1,31 @@
...
-#
-#   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
-#   Character.create(name: 'Luke', movie: movies.first)
-# require 'csv'
-
-# csv_text = File.read(Rails.root.join('lib', 'seeds', 'real_estate_transactions.csv'))
-# puts csv_text
\ No newline at end of file
+require 'csv'
+
```
Comment (ghost) [inline db/seeds.rb:2]: Can we wrap this file to create the rails csv_load:all task? Might have to move it to lib/tasks

```
namespace :csv_load do

desc "Import customers from CSV"
task customers: :environment do
  # CSV code here 
    end
  end
end
```

**Review by ghost — COMMENTED** [review]
@joemecha this is great, thank you for setting all the models and csv code up
i made a couple of comments for us to think about, but the code itself looks good so i'm ok with merging this

File: db/seeds.rb (hunk 2 above)
Comment (joemecha) [reply db/seeds.rb:2]: @koshea9 yes, I think this is our next step so we can get those custom rake commands going

**Review by brueck1988 — APPROVED** [review]

**Pull request merged**